### DISCUSSION TITLE
2022-47: Databend 文档的 i18n 实践

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#390] [Announcements]
# 2022-47: Databend 文档的 i18n 实践

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-47/

**Comment by ycjcl868** [comment]
求问图一、图二是用啥工具画出来的？

**Reply by Xuanwo** [reply]
图一用的是 https://github.com/tldraw/tldraw ，图二是 Crowdin 自己的 Pipeline 编辑器
